modules/fedimint-lnv2-tests/tests/mock.rs

```diff
@@ -89,9 +89,9 @@ impl GatewayConnection for MockGatewayConnection {
     ) -> Result<Option<RoutingInfo>, GatewayError> {
         Ok(Some(RoutingInfo {
             public_key: self.keypair.public_key(),
-            send_fee_default: PaymentFee::one_percent(),
-            send_fee_minimum: PaymentFee::half_of_one_percent(),
-            receive_fee: PaymentFee::half_of_one_percent(),
+            send_fee_default: PaymentFee::ten_promille_plus_50_sats(),
```

Comment:
Not necessary to change, but these probably could be `const` as `PaymentFee::TEN_PROMILLE_...` etc.

Reply:
Done.